modules/local/predict_possible_neoepitopes.nf

```diff
@@ -1,34 +1,22 @@
-// Import generic module functions
-include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
-
-params.options = [:]
-options        = initOptions(params.options)
-
 process PREDICT_POSSIBLE_NEOEPITOPES {
```

Comment:
Maybe include CLASS1 in the process name to get it in line with PREDICT_POSSIBLE_CLASS2_NEOEPITOPES

Reply:
Same here